<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddActivity</class>
 <widget class="QMainWindow" name="AddActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 31, 31);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,10,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5,2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item alignment="Qt::AlignLeft|Qt::AlignTop">
           <widget class="QPushButton" name="pushButton_profile">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
	margin-top:10px;
	margin-left:10px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
            </property>
            <property name="text">
             <string>PROFILE</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>30</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>ACTIVITY</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>TYPE</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QComboBox" name="comboBox_type">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>53</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox {
	color: #CCCCCC;
    background-color: #333333;
    border: 1px solid #666666;
    padding: 2px 8px;
}

QComboBox::drop-down {
    border: none;
}


QComboBox::item {
    background-color: #666666;
	color: rgb(255, 255, 255);
}

QComboBox::item:Hover {
    background-color: #666666;
	color: rgb(255, 255, 255);
}

QComboBox::item:selected {
    background-color: #444444;
	color: rgb(255, 255, 255);
}
</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>VALUE</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QComboBox" name="comboBox_value">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>53</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox {
    color: #CCCCCC;
    background-color: #333333;
    border: 1px solid #666666;
    padding: 2px 8px;
}

QComboBox::drop-down {
    border: none;
}

QComboBox::down-arrow {
    image: url(down-arrow.png);
}

QComboBox::item {
    background-color: #666666;
}

QComboBox::item:selected {
    background-color: #444444;
}
</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="1,5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>DESCRIPTION : </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QLineEdit" name="lineEdit_description">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    border: 2px solid #333333; 
    border-radius: 10px;
    padding: 8px 12px;
    font-size: 16px;
    color: #FFFFFF; 
    background-color: #555555; 
    height: 40px; 
}

QLineEdit:focus {
    border-color: #4CAF50; 
    background-color: #666666; 
}

QLineEdit::placeholder {
    color: #AAAAAA; 
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QPushButton" name="pushButton">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
              </property>
              <property name="text">
               <string>ADD ACTIVITY</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
